tests/integration/test_nitrate.py

```diff
@@ -105,3 +106,32 @@ def test_import_manual_proposed(self):
         self.assertEqual(result.output.strip(), "")
         fmf_node = Tree(self.tmpdir).find("/import_case")
         self.assertEqual(fmf_node, None)
+
+
+class NitrateImportAutomated(RequreTestCase):
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(EXAMPLES / "convert", self.tmpdir)
+        self.cwd = os.getcwd()
+
+    def tearDown(self):
+        os.chdir(self.cwd)
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_basic(self):
+        os.chdir(self.tmpdir)
+        files = os.listdir()
+        self.assertIn("Makefile", files)
+        self.assertNotIn("main.fmf", files)
+        self.assertNotIn("test.md", files)
+        runner = CliRunner()
+        result = runner.invoke(
+            tmt.cli.main, [ "test", "import", "--nitrate"])
+        self.assertEqual(result.exit_code, 0)
+        files = os.listdir()
+        self.assertIn("Makefile", files)
+        self.assertIn("main.fmf", files)
```

Comment:
Could you assert that import put correct content in the file as well?

Reply:
theoretically yes, but depends how much you want to check it, theoretically I can copy current content of main.fmf and main.md and use it as reference, but finally it needs someone who understand what there could be, especially, what should be there from perspective, what has higher priority nitrate/Makefile, for bugs, requires or whatever and potentially you can find some issue right now.

@lukaszachy So I think that we should merge it as it is and whoever want can extent it by more asserts what makes sense.